src/tool_cb_wrt.c

```diff
@@ -364,7 +364,10 @@ size_t tool_write_cb(char *buffer, size_t sz, size_t nmemb, void *userdata)
 
   if(config->nobuffer) {
     /* output buffering disabled */
-    int res = fflush(outs->stream);
+    int res;
+    do {
+      res = fflush(outs->stream);
+    } while(res && errno == SOCKEINTR);
```

Comment:
```suggestion
        /* !checksrc! disable ERRNOVAR 1 */
        } while(res && errno == EINTR);
```
Since we're operating on file I/O, wouldn't we rather want the non-socket `EINTR`?